packages/workflow/src/LoggerProxy.ts

```diff
@@ -0,0 +1,45 @@
+import { 
+	ILogger, 
+	LogTypes,
+} from './Interfaces';
+
+
+let instance: ILogger | undefined;
+
+export function init(logger: ILogger) {
+	instance = logger;
```

Comment:
To keep consistency with the getters:

```suggestion
let logger: ILogger | undefined;

export function init(loggerInstance: ILogger) {
	logger = loggerInstance;
```